hack/make-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# set vars 
+if [ -n "$1" ]; then
+    DOCKER_REPO=$1
+else
+    echo "Must provide a docker repo"
+    exit 
+fi
+
+if [ -n "$2" ]; then
```

Comment:
instead of relying on positional args, you can simply do:

    TAG="${TAG?TAG env variable must be specified}"

and you don't deal with ifs, positional arg parsing etc.